src/styles.css

```diff
@@ -90,34 +99,27 @@ p {
 ul {
     text-align: center;
     border-radius: 4px;
-    border: 1px darkgrey;
-    padding: 20px;
+    border: 1px lightcyan;
+    padding: 0;
+    margin: 0;
     display: flex;
     flex-wrap: wrap;
     flex-direction: row;
-    justify-content: space-between;
+    justify-content: space-evenly;
     list-style: none;
```

Comment:
Ik kan helaas de npm niet runnen. Krijg errors. Hierdoor kan ik het landenoverzicht niet zien. Misschien kun je hier list style vervangen door list-style-type: none. Wellicht helpt dit om de komma's weg te halen :)
Wat je misschien ook kan doen is in app.js de inner html aanpassen door : 
const countriesList = document.getElementById("country-list");
        countriesList.innerHTML = countryMap.join('');

Reply:
hmm nee werkt ook niet. Al verschillende aanvliegroutes geprobeerd. Ik ga de hulplijnen inschakelen ;-)